### ISSUE TITLE
How to assert a website page that is outside my app? 

### ISSUE THREAD

**Issue by americapires1** [calabash/calabash-android#622]
Hi, 
I'm using calabash android. 
On my app I have a button that directs me to O2 website. 
Could anyone please let me know if there a way to assert that Im on the O2 website?

I have been googling this but unsuccessfully. 

Thanks 
America

**Comment by sherbhachu** [comment]
You could just check the URL? or explicitly check for an element that you
know appears once you are on the O2 site?

**Comment by americapires1** [comment]
I have tried that before and once Im directed to the O2 website (outside my app), Im unable to query it. however I've tried to to go the O2 website and inspect some elements and used a text element from the O2 website and query it, e.g. 
query("Textview text:'O2 | Mobile Phones, Mobile Broadband and Sim Only Deals on O2'") 
and yet nothing is returned from the query. 

I have also tried;
query("Textview text:'O2 | Mobile Phones, Mobile Broadband and Sim Only Deals on O2'" , :text).first  
and it return nil.
